client/flutter/pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dependencies:
   fl_chart: ^0.8.7
 
   intl: ^0.16.0
+  intl_utils: ^1.2.2
```

Comment:
Since this is only used by the build process I think this dependency can be moved under `dev_dependencies` in the `pubspec.yaml`.